x/streamer/types/codec.go

```diff
@@ -17,6 +18,12 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgUpdateStream{}, "streamer/UpdateStream", nil)
 	cdc.RegisterConcrete(&MsgUpdateParams{}, "streamer/UpdateParams", nil)
 	cdc.RegisterConcrete(Params{}, "streamer/Params", nil)
+
+	// Register legacy proposal types for backward compatibility with existing state
+	cdc.RegisterConcrete(&CreateStreamProposal{}, "streamer/CreateStreamProposal", nil)
+	cdc.RegisterConcrete(&TerminateStreamProposal{}, "streamer/TerminateStreamProposal", nil)
+	cdc.RegisterConcrete(&UpdateStreamDistributionProposal{}, "streamer/UpdateStreamDistributionProposal", nil)
+	cdc.RegisterConcrete(&ReplaceStreamDistributionProposal{}, "streamer/ReplaceStreamDistributionProposal", nil)
```

Comment:
I think those 3 we're never used before.. if that's the case not sure we need to register them and create the proposal_distribution.go file